### ISSUE TITLE
[FEATURE] Add baritone integration?

### ISSUE THREAD

**Issue by sqlerrorthing** [CCBlueX/LiquidBounce#4892]
### LiquidBounce Branch

Nextgen

### Describe your feature request.

I would like to see integration with [Baritone](https://github.com/cabaletta/baritone), is there any point in it, or do you think the client doesn't need it at all?

**Comment by Razzy52** [comment]
You can just add it into the mods folder?

**Comment by sqlerrorthing** [comment]
> You can just add it into the mods folder?

You misunderstood me, I'm suggesting to add exactly baritone integration like with ViaFabricPlus
Intagration like in [Meteor Client](https://github.com/MeteorDevelopment/meteor-client), there is a whole block in the gui to customize baritone from under the client

**Comment by 1zun4** [comment]
Yes, I think integrating Baritone to LiquidBounce would be great. Quite a few features could also make use of it...

**Comment by sqlerrorthing** [comment]
up to tops

**Comment by unlovingbottom** [comment]
stale?
